spv/sync.go

```diff
@@ -70,8 +71,22 @@ type Syncer struct {
 	currentLocators   []*chainhash.Hash
 	locatorGeneration uint
 	locatorMu         sync.Mutex
+
+	// Holds all of the potential callbacks to update the grpc request
+	notificationCallbacks *NtfnsCallbacks
+}
+
+// NtfnsCallbacks struct to contain all of the upcoming callbacks that will
+// be used to update the rpc streams for syncing.
+type NtfnsCallbacks struct {
+	// Update when the wallet becomes synced or unsynced
```

Comment:
This is also a doc comment and should begin with the identifier it is documenting.